alexandrie/src/utils/request_log.rs

```diff
@@ -22,7 +22,14 @@ impl<State: Clone + Send + Sync + 'static> Middleware<State> for RequestLogger {
         let res = next.run(req).await;
         let elapsed = start.elapsed().as_millis();
         let status = res.status();
-        info!("--> {} {} {} {}ms", method, path, status, elapsed,);
+        let mut resp_msg = "OK".into();
+        if let Some(err) = res.error() {
+            resp_msg = err.to_string()
+        }
```

Comment:
I would write it like this, to avoid the `mut` on `resp_msg`:
```suggestion
        let resp_msg = if let Some(err) = res.error() {
            err.to_string()
        } else {
            "OK".to_string()
        };
```

Reply:
Thanks for your suggestion. I accept this suggestion. But I think it's not a beautiful syntax code.So I just use a `{}` code block to enhance expression. Like this:

* your suggestion code
```rust
let resp_msg = if let Some(err) = res.error() {
    err.to_string()
} else {
    "OK".to_string()
};
```

* my suggestion code
```rust
let mut resp_msg = {
    if let Some(err) = res.error() {
        err.to_string()
    } else {
        "OK".into()
    }
};
```